internal/server/kong/ws/config/compat/compatibility_table.go

```diff
@@ -404,23 +404,6 @@ var (
 				},
 			},
 		},
-		{
-			Metadata: config.ChangeMetadata{
-				ID:       config.ChangeID("P114"),
-				Severity: config.ChangeSeverityError,
-				Description: standardPluginFieldsMessage("response-ratelimiting",
-					[]string{"redis_username"}, "2.8", false),
-				Resolution: standardUpgradeMessage("3.0"),
-			},
-			SemverRange: versionsPre280,
-			Update: config.ConfigTableUpdates{
-				Name: "response-ratelimiting",
-				Type: config.Plugin,
-				RemoveFields: []string{
-					"redis_username",
-				},
-			},
-		},
```

Comment:
Maybe we should leave a small comment for ourselves that P114 was removed due to duplication.

```
// P114 was removed due to duplication and inaccuracies for associated version upgrade
```

Something like that; I am sure that can be worded a lot better.

Reply:
Not by me :P